lib/homepage_constraint.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class HomePageConstraint
+class HomepageConstraint
```

Comment:
Changing the constant name could potentially break plugins that rely on it. Would it be safer to rename the filename instead?